src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "CloudWatch fallback"
+metaDescription: This doc guides you through shipping your telemetry data with Cloudwatch as a fail-safe option.
+freshnessValidatedDate: never
+---
+
+You can ship New Relic Lambda monitoring data in a few different ways depending on your needs. The option that provides the strongest performance of your function and the strongest fail-safe is the CloudWatch fallback option. This means that you will ship telemetry to New Relic through the agent and use the extension to enhance the telemetry collection but, if there's a problem with the extension, you will still receive logs and payloads in CloudWatch.  
+
+While this offers the strongest fail-safe, it's important to note that the AWS CloudWatch service can generate a lot of data. Keep data ingest in mind when thinking about costs and which New Relic pricing plan you choose. 
+
+Benefits of the CloudWatch fallback:
+
+* **Independent execution**: Captures all logs from CloudWatch independently of your function's execution. There's less chance it will interfere with the normal operation, or impact the invocation duration of your function.
+* **Works with secured networks**: Useful for functions in a VPC that can't have outbound traffic to New Relic.
+* **Enables enhanced monitoring:  Needed for other forms of infra and database telemetry. To learn more, see our [Amazon RDS Enhanced Monitoring integration docmentation](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-rds-enhanced-monitoring-integration/).
+
```

Comment:
Best practices:

* Prior to updating to a newer layer version, consider that the latest layer might introduce breaking changes in the Extension or a third party dependency.
* It’s a good idea to pin a layer version that is known to work well for your function, and only update to a newer layer version as needed to resolve bugs, obtain new features / security patches, or before an agent EOL.
* Carefully review all changes and release notes and test in a lower environment (dev, staging) before deploying a new layer to production.

Reply:
I edited it to this, does this work? 

Here are some best practices before updating your New Relic Lambda layer:

    * Before you update to a newer layer version, keep in mind that the latest layer might introduce changes that break the extension or a third party dependency.
    * Pin a layer version that is known to work well for your function and only update to a newer layer version as needed to resolve bugs, obtain new features and security patches, or because an agent has EOL'd.
    * Carefully review all changes and release notes and test in a developer or staging environment before deploying a new layer to production.

Reply:
A developer environment is typically referred to as the dev environment.

Reply:
This has been changed!